controllers/sendUser.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
+  res.send(req.user);
```

Comment:
also where did `req.user` come from ?

Reply:
It is in the request, put there by passport and cookie-session. It's always on the request unless the user logs out. It's serialized and deserialised using Passport.